tests/docker/Dockerfile

```diff
@@ -75,7 +75,7 @@ RUN apt update && \
 # install go
 RUN mkdir -p /usr/local/go/ && \
     bash -c 'if [[ $(uname -m) = "aarch64" ]]; then ARCHID="arm64"; else export ARCHID="amd64"; fi && \
-    curl -sSLf --retry 3 --retry-connrefused --retry-delay 2 https://golang.org/dl/go1.17.linux-${ARCHID}.tar.gz | tar -xz -C /usr/local/go/ --strip 1'
+    curl -sSLf --retry 3 --retry-connrefused --retry-delay 2 https://golang.org/dl/go1.18.2.linux-${ARCHID}.tar.gz | tar -xz -C /usr/local/go/ --strip 1'
```

Comment:
```suggestion
    curl -sSLf --retry 3 --retry-connrefused --retry-delay 2 https://golang.org/dl/go1.18.6.linux-${ARCHID}.tar.gz | tar -xz -C /usr/local/go/ --strip 1'
```

latest of [go1.18](https://go.dev/dl/) is now 1.8.6

Reply:
Done. My apologies for taking so long to get to this.